README.md

```diff
@@ -109,43 +109,65 @@ CLI for kubelet CRI
 
 #### Examples
 
-- Run podsandbox with config file
-
-`crictl sandbox run pod.json`
-
-example of config file for podsandbox `pod.json`:
+- Run sandbox with config file
 
 ```
+# cat sandbox-config.json
 {
     "metadata": {
-        "name": "podsandbox-1",
+        "name": "nginx-sandbox",
         "namespace": "default",
-        "attempt": "2",
+        "attempt": 1,
         "uid": "hdishd83djaidwnduwk28bcsb"
     },
     "linux": {
     }
 }
+# crictl sandbox run sandbox-config.json
+9b542bfe8f93eb2d726d0f7b619f253c18858006aa53023e392e138b0be6301c
 ```
-- Create container in a sandbox with container config file
-
-`crictl container create --pod podid --connfig container.json` (replace `podid` with id of a ready sandbox)
 
-example of config file for container `container.json`:
+- Create container in a sandbox with config file
 
 ```
-    {
+# cat sandbox-config.json
+{
     "metadata": {
-        "name": "container-1"
-    },
-    "image":{
-        "image": "busybox"
+        "name": "nginx-sandbox",
+        "namespace": "default",
+        "attempt": 1,
+        "uid": "hdishd83djaidwnduwk28bcsb"
     },
-    "command": [
-        "sh",
-        "-c"
-    ],
     "linux": {
     }
 }
+# cat container-config.json
+{
+  "metadata": {
+      "name": "busybox"
+  },
+  "image":{
+      "image": "busybox"
+  },
+  "command": [
+      "top"
+  ],
+  "linux": {
+  }
+}
```

Comment:
There are much more fields in container config, this is just a simple example. We don't need to add all fields in it.